lib/mail_form/delivery.rb

```diff
@@ -15,7 +15,12 @@ module Delivery
       class_attribute :mail_appendable
       self.mail_appendable = []
 
-      if respond_to?(:before_deliver) && respond_to?(:after_deliver)
+      # not_spam? simply returns false, but when used as a callback, we have to throw(:abort)
+      # to halt the callback chain, beginning with Rails 5.1.
+      if ActionPack.respond_to?(:version) && ActionPack.version >= Gem::Version.new('5.1')
```

Comment:
mail_form supports Rails 3.2. It didn't have `version` back then: https://github.com/rails/rails/blob/3-2-stable/actionpack/lib/action_pack/version.rb